packages/indexer/lib/index.js

```diff
@@ -249,7 +254,11 @@ class CellCollector {
     } else {
       outPoints = typeOutPoints;
     }
-    return outPoints;
+    if (this.order === "desc") {
+      return outPoints.reverse();
```

Comment:
Are we sure this is enough? Don't we need to do a query from the other direction at rocksdb side? I recall there are a lot of code to support order query in ckb-indexer

Reply:
I first came out with the same way you suggesting here, which would query the rocksdb with order parameter via rust bindings directly, but it seems quite ok to reserve the default query results, which is already well-ordered by block number  in ascending order.

Reply:
Yep, it seems `not` ok after further communication. gonna fix it.